README.md

```diff
@@ -24,6 +24,7 @@ Our license information can be found [here](docs/LicenseInfo.md) and our license
     - [How Akita Helps](#how-akita-helps)
     - [Blog Posts on Our Process](#blog-posts-on-our-process)
     - [Why 'Akita'?](#why-akita)
+    - [Deleting Akita Data](#deleting-akita-data)
```

Comment:
this doesn't match the name/link of the data section `Akita Data` which would be `#akita-data`